.github/workflows/no-response.yml

```
@@ -18,13 +20,15 @@ jobs:
       contents: read
       issues: write
     steps:
-      - uses: lee-dohm/no-response@9bb0a4b5e6a45046f00353d5de7d90fb8bd773bb # v0.5.0
+      - uses: MBilalShafi/no-response-add-label@629add01d7b6f8e120811f978c42703736098947
```

The GitHub Action update bot will add a comment about the version, so we might as well add it upfront, e.g. https://github.com/mui/mui-x/pull/11430/files

```suggestion
      - uses: MBilalShafi/no-response-add-label@629add01d7b6f8e120811f978c42703736098947 # v0.0.6
```